src/core/storage/storagekeys.js

```diff
@@ -30,7 +30,7 @@ export default {
   SESSIONS_OPT_IN: 'sessions-opt-in',
   VERTICAL_PAGES_CONFIG: 'vertical-pages-config',
   LOCALE: 'locale',
-  SORT_BYS: 'sort-bys',
+  SORT_BYS: 'sort-bys', // Cut over to new global storage
```

Comment:
nit: this almost reads like a todo when really it's just a note. How about '// Has been cut over to the new global storage'?

Reply:
I like that, will update in the next commit!